netbox/project-static/src/search.ts

```diff
@@ -9,36 +10,59 @@ function quickSearchEventHandler(event: Event): void {
   const quicksearch = event.currentTarget as HTMLInputElement;
   const inputgroup = quicksearch.parentElement as HTMLDivElement;
   if (isTruthy(inputgroup)) {
-    if (quicksearch.value === "") {
-      inputgroup.classList.add("hide-last-child");
+    if (quicksearch.value === '') {
+      inputgroup.classList.add('hide-last-child');
     } else {
-      inputgroup.classList.remove("hide-last-child");
+      inputgroup.classList.remove('hide-last-child');
     }
   }
 }
 
+/**
+ * Update the Export View link to add the Quick Search parameters.
+ * @param event
+ */
+
+function handleQuickSearchParams(event: Event): void {
+  console.log('getParams');
+  const quickSearchParameters = event.currentTarget as HTMLInputElement;
+  const link = document.getElementById('current_view') as HTMLLinkElement;
+  if (quickSearchParameters != null) {
+    const search_parameter = `q=${quickSearchParameters.value}`;
+    const linkUpdated = link?.href + '&' + search_parameter;
```

Comment:
This appears to append successive queries, e.g. `?q=foo&q=bar`. Any previous query should be replaced by the new one.